src/main/scala/org/broadinstitute/dsde/workbench/sam/api/RejectionHandlers.scala

```diff
@@ -0,0 +1,22 @@
+package org.broadinstitute.dsde.workbench.sam
+package api
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives.complete
+import akka.http.scaladsl.server.{MalformedRequestContentRejection, RejectionHandler, RequestEntityExpectedRejection}
+import org.broadinstitute.dsde.workbench.model.ErrorReport
+import org.broadinstitute.dsde.workbench.model.ErrorReportJsonSupport.ErrorReportFormat
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+
+object RejectionHandlers {
+
+  def termsOfServiceErrorReport(tosUrl: String) = ErrorReport(StatusCodes.Forbidden, s"You must accept the Terms of Service in order to register. See ${tosUrl}")
+
+  def termsOfServiceRejectionHandler(tosUrl: String) = RejectionHandler.newBuilder().handle {
+    case RequestEntityExpectedRejection =>
+      complete(StatusCodes.Forbidden, termsOfServiceErrorReport(tosUrl))
+    case MalformedRequestContentRejection(_, _) =>
+      complete(StatusCodes.Forbidden, termsOfServiceErrorReport(tosUrl))
```

Comment:
nit: combine these into one case, since they do the same thing:
```suggestion
    case RequestEntityExpectedRejection | MalformedRequestContentRejection(_, _) =>
      complete(StatusCodes.Forbidden, termsOfServiceErrorReport(tosUrl))
```